start-server.sh

```diff
@@ -15,14 +15,30 @@
 # limitations under the License.
 
 #provide the path to Node directory before starting the run
-#NODE_PATH=/opt/local/node-v6.10.0-linux-x64/
-if [ "x$NODE_PATH" == "x" ]; then
-	echo "Node path is not set"
+export PATH=/opt/local/node-v6.10.0-linux-x64/bin:$PATH
```

Comment:
This needs to change as per the environment or sometime not needed since 'node' is installed in a standard path.